### PR TITLE
Include doc on enabling GCE cloud provider for custom clusters

### PR DESCRIPTION
**Pull request by catherineluse**
This PR is intended to pull information from this gist https://gist.github.com/thxCode/3d6805a904fdb34a03010c0908e25bed into the Rancher documentation so that it can be used to enable the GCE cloud provider.

### CODE REVIEW

File: content/rancher/v2.x/en/cluster-provisioning/rke-clusters/cloud-providers/gce/_index.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+title: Setting up the Google Compute Engine Cloud Provider
+weight: 3
+---
+
+In this section, you'll learn how to enable the Google Compute Engine (GCE) cloud provider for provisioning custom clusters in Rancher. A custom cluster is one in which Rancher installs Kubernetes on existing nodes.
+
+These steps are required for provisioning GCE Kubernetes clusters that include Windows nodes.
```
Comment (thxCode) [inline content/rancher/v2.x/en/cluster-provisioning/rke-clusters/cloud-providers/gce/_index.md:8]: I think the GCE cloud provider should be suitable for the Linux cluster too.

File: content/rancher/v2.x/en/cluster-provisioning/rke-clusters/cloud-providers/gce/_index.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        [Global]
+        project-id=<your project ID, optional>
+        network-name=<your network, optional if using default network>
+        subnetwork-name=<your subnetwork of the above network, optional if using default network>
+        node-instance-prefix=<your instance group name/your instance name specific prefix, required>
+        node-tags=<your network tags, must patch one or some tags, required>
+    services:
+        kube_controller:
+        extra_args:
+            configure-cloud-routes: true # we need to allow
```
Comment (thxCode) [inline content/rancher/v2.x/en/cluster-provisioning/rke-clusters/cloud-providers/gce/_index.md:53]: There are some problems with the content format above, some spaces should be indented:
``` yaml
...
rancher_kubernetes_engine_config:
  cloud_provider: // indent 2 spaces than `rancher_kubernetes_engine_config`
    name: gce
    customCloudProvider: |- // content should be indented
      [Global]
      project-id=<your projcet ID, optional>
      network-name=<your network, optional if using default network>
      subnetwork-name=<your subnetwork of the above network, optional if using default network>
      node-instance-prefix=<your instance group name/your instance name specific prefix, required>
      node-tags=<your network tags, must patch one or some tags, requried>
  services: // indent 2 spaces than `rancher_kubernetes_engine_config`
    kube_controller: 
      extra_args: // indent 2 spaces than `kube_controller`
        configure-cloud-routes: true # we need to allow the cloud provider configure the routes for the hosts
...
```

File: content/rancher/v2.x/en/cluster-provisioning/rke-clusters/cloud-providers/gce/_index.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+        [Global]
+        project-id=<your projcet ID, optional>
+        network-name=<your network, optional if using default network>
+        subnetwork-name=<your subnetwork of the above network, optional if using default network>
+        node-instance-prefix=<your instance group name/your instance name specific prefix, required>
+        node-tags=<your network tags, must patch one or some tags, required>
+    network: 
+        options: 
+        calico_cloud_provider: "gce"
+        plugin: "calico"
```
Comment (thxCode) [inline content/rancher/v2.x/en/cluster-provisioning/rke-clusters/cloud-providers/gce/_index.md:32]: There are some problems with the content format above, some spaces should be indented:
```yaml
...
rancher_kubernetes_engine_config:
  cloud_provider: // indent 2 spaces than `rancher_kubernetes_engine_config`
    name: gce
    customCloudProvider: |- // content should be indented
      [Global]
      project-id=<your projcet ID, optional>
      network-name=<your network, optional if using default network>
      subnetwork-name=<your subnetwork of the above network, optional if using default network>
      node-instance-prefix=<your instance group name/your instance name specific prefix, required>
      node-tags=<your network tags, must patch one or some tags, requried>
  network: // indent 2 spaces than `rancher_kubernetes_engine_config`
    options: 
      calico_cloud_provider: "gce" // indent 2 spaces than `options`
    plugin: "calico"
...
```

**Comment by catherineluse** [comment]
I accidentally merged this PR. I'm going to make another PR to respond to this feedback

File: content/rancher/v2.x/en/cluster-provisioning/rke-clusters/cloud-providers/gce/_index.md (hunk 1 above)
Comment (catherineluse) [reply content/rancher/v2.x/en/cluster-provisioning/rke-clusters/cloud-providers/gce/_index.md:8]: Yes, Sowmya confirmed it, I will change it.

**Comment by catherineluse** [comment]
These errors have been fixed in the following commits:

https://github.com/rancher/docs/commit/ca2386656c0cdeb9f57a5ba3f6158b258900a85a
https://github.com/rancher/docs/commit/ae7a2d5d365cb3a44fb0b9172a7d2de922ffbdc9

**Pull request merged**